### DISCUSSION TITLE
Baseline values only for carryover model

### DISCUSSION THREAD

**Discussion by rafpinter** [google/lightweight_mmm#99] [Q&A] [answered]
Hi everyone!

First, thanks for making this amazing open source library.

I have noticed that only my carryover models present relevant baseline values on the attribution plot. Any explanation/idea on why such thing happens?

Carryover:
![image](https://user-images.githubusercontent.com/62558186/199805695-331ffadf-165a-40f2-90ff-896a20b2897f.png)

Adstock (and hill-adstock also showed this behavior):
![image](https://user-images.githubusercontent.com/62558186/199805621-a26acaf2-1ab2-45c7-a50c-a4a8f2e0076e.png)

Thanks!

**Comment by michevan** [comment, marked as answer]
Hi @rafpinter, thanks for sharing the plots! 

The answer here will absolutely depend on your data, so I'm not sure just based on these plots, but I'd recommend taking a look at the prior and posterior distributions for the shape parameters in all three of your models (these are things like ad_effect_retention_rate, peak_effect_delay, and exponent for the carryover model, or exponent and lag_weight for the adstock model). See if any of the posterior distributions look wildly different from the priors, and also if they look reasonable or unreasonable for those channels, and if they look different from the priors and reasonable then you could try adjusting the priors a bit using [custom priors](https://lightweight-mmm.readthedocs.io/en/latest/custom_priors.html). My guess would be that lag_weight might be the culprit, since it's used in both adstock and hill_adstock but not in the carryover model. 

Good luck, and please let us know how it goes!
